src/develop/imageop.c

```diff
@@ -1021,49 +1021,29 @@ static gboolean _rename_module_key_press(GtkWidget *entry, GdkEventKey *event, d
 
 static void _iop_gui_rename_module(dt_iop_module_t *module)
 {
-  const int bs = DT_PIXEL_APPLY_DPI(12);
-  gint px = 0, py = 0;
-
   dt_iop_gui_rename_module_t *d = (dt_iop_gui_rename_module_t *)calloc(1, sizeof(dt_iop_gui_rename_module_t));
   d->module = module;
-  GtkWidget *window = dt_ui_main_window(darktable.gui->ui);
 
   GList *childs = gtk_container_get_children(GTK_CONTAINER(module->header));
```

Comment:
childs _does_ need to be g_list_free'd (and might as well be renamed children then)